Source/GitSourceControl/Private/GitSourceControlOperations.cpp

```diff
@@ -530,15 +531,16 @@ bool FGitRevertWorker::Execute(FGitSourceControlCommand& InCommand)
 			// revert any changes in working copy (this would fails if the asset was in "Added" state, since after "reset" it is now "untracked")
 			// may need to try a few times due to file locks from prior operations
 			bool CheckoutSuccess = false;
-			for (int32 i = 0; i < 10; ++i)
+			int32 Attempts = 0;
+			while( --Attempts >= 0 )
```

Comment:
Attempts is init to 0 instead of 10, and I think it's better to go with post increment and > than pre and >=

Reply:
Done